preview.go

```diff
@@ -68,6 +68,57 @@ func (o Output) MarshalJSON() ([]byte, error) {
 	})
 }
 
+// ValidatePrebuilds will iterate over each preset, validate the inputs as a set,
+// and attach any diagnostics to the preset if there are issues. This must be done
+// because prebuilds have to be valid without user input.
+//
+// This will only validate presets that have prebuilds configured and have no
+// existing error diagnostics. This should only be used when doing Template
+// Imports as a protection against invalid presets. A template can still be
+// usable if it's presets are invalid. It just cannot be used for prebuilds.
```

Comment:
`A template can still be usable if it's presets are invalid. It just cannot be used for prebuilds.`
I'm a bit confused with this sentence, is this to say that it is still possible to create regular (non-prebuilt) workspaces? And only in case a preset is invalid, it cannot be used for creating prebuilt workspaces?

Reply:
Makes sense 👍 nit: updating the comment, maybe something like:
```
A preset doesn't need to specify all required parameters, as users can provide the remaining values when creating a workspace. However, since prebuild creation has no user input, presets used for prebuilds must provide all required parameter values.
```